bindings/python/iota_sdk/client/client.py

```diff
@@ -406,7 +406,7 @@ def build_and_post_block(self,
             End of the input range
         inputs : Array of Inputs
             Inputs to use
-        output : SendAmountParams
```

Comment:
Undo all the changes in this file and just do `output: Any  # TODO: https://github.com/iotaledger/iota-sdk/issues/129`